artemis/modules/admin_panel_login_bruter.py

```diff
@@ -0,0 +1,247 @@
+    """
+
+    identity = "admin_panel_login_bruter"
+    filters = [{"type": TaskType.URL.value}]
+
+    USERNAMES = ["admin"]
+
+    LOGIN_FAILED_MSGS = [
+        "Please enter the correct username and password for a staff account. "
+        "Note that both fields may be case-sensitive.",
+        "Unrecognized username or password. Forgot your password?",
+        "Username and password do not match or you do not have an account yet.",
+        "Invalid credentials",
+        "The login is invalid",
+        "Login failed",
+        "not seem to be correct",
+        "Access denied",
+        "Cannot log in",
+        "login details do not seem to be correct",
+        # rate limit
+        "failed login attempts for this account",
+        # pl_PL
+        "Podano błędne dane logowania",
+        "Wprowadź poprawne dane",
+        "Nieprawidłowa nazwa użytkownika lub hasło",
+        "Nazwa użytkownika lub hasło nie jest",
+        "Złe hasło",
+        "niepoprawne hasło",
+        "Błędna nazwa użytkownika",
+    ]
+
+    def run(self, task: Task) -> None:
+        url = task.get_payload(TaskType.URL)
+        url_parsed = urllib.parse.urlparse(url)
+
+        is_root_url = url_parsed.path in ["", "/"]
+
+        if (
+            not is_root_url
+            and not url_parsed.path.endswith("/")
+            and not any([item in url.lower() for item in ["login", "admin", "cms", "backend", "panel", "index.php"]])
+        ):
+            self.db.save_task_result(
+                task=task,
+                status=TaskStatus.OK,
+                status_reason=f"URL {url} doesn't look like an admin panel URL",
+                data=None,
+            )
+            return
+
+        if "admin" in url and not url.endswith("/admin") and "admin" in task.get_payload("found_urls"):
+            self.db.save_task_result(
+                task=task,
+                status=TaskStatus.OK,
+                status_reason=f"Requested to brute {url}, but /admin is also on the list of found URLs - skipping "
+                "brute-forcing of this url so that we don't duplicate work",
+                data=None,
+            )
+            return
+
+        credentials = self._brute(url)
+
+        if credentials:
+            username, password = credentials
+
+            self.db.save_task_result(
+                task=task,
+                status=TaskStatus.INTERESTING,
+                status_reason=f"Found working credentials for {url}: username={username}, password={password}",
+                data=credentials,
+            )
+        else:
+            self.db.save_task_result(task=task, status=TaskStatus.OK, status_reason=None, data=None)
+
+    def _brute(self, url: str) -> Optional[Tuple[str, str]]:
+        working_credentials = []
+        for username in self.USERNAMES:
+            for password in get_passwords_for_url(url):
+                driver = AdminPanelLoginBruter._get_webdriver()
```

Comment:
Would not it be better to get only one webdriver for whole `_brute` function and just drop old cookies every 
credential? Now it will be resource consuming